cmd/kubeadm/app/cmd/upgrade/plan.go

```diff
@@ -109,27 +288,20 @@ func runPlan(flags *planFlags, args []string) error {
 		// in the human readable output if it did so
 		plan.ConfigVersions = configVersionStates
 
-		printUpgradePlan(&up, plan, unstableVersionFlag, isExternalEtcd, os.Stdout)
+		printUpgradePlan(&up, plan, unstableVersionFlag, isExternalEtcd, os.Stdout, printer)
 	}
 
 	// Finally, print the component config state table
-	printComponentConfigVersionStates(configVersionStates, os.Stdout)
+	printComponentConfigVersionStates(configVersionStates, os.Stdout, printer)
 
+	// Add a newline in the end of this output to leave some space to the next output section
+	klog.V(1).Infoln()
```

Comment:
i think the new line should be at the end of the previous output that is being printed instead of being added here explicitly.

also having v(1) here seems strange as the new line will be only visible with --v=1.

Reply:
I prefer to remove this directly.